src/styles/settings/colors/reserved-color-scale.css

```diff
@@ -1,10 +1,14 @@
 :root {
   --ec-reserved-color-error: 0, 72%, 56%;
-  --ec-reserved-color-error-hover: 0, 72%, 46%;
+  --ec-reserved-color-error-hover: 0, 72%, 44%;
+  --ec-reserved-color-error-light: 0, 59%, 94%;
   --ec-reserved-color-info: 205, 100%, 56%;
-  --ec-reserved-color-info-hover: 205, 100%, 46%;
+  --ec-reserved-color-info-hover: 205, 100%, 37%;
+  --ec-reserved-color-info-light: 205, 58%, 94%;
   --ec-reserved-color-success: 162, 80%, 41%;
-  --ec-reserved-color-success-hover: 162, 80%, 31%;
+  --ec-reserved-color-success-hover: 162, 80%, 26%;
+  --ec-reserved-color-success-light: 164, 28%, 92%;
```

Comment:
I think H should be 162, just like the two other success colors. L should be 94% just like the other 3 light colors.

Reply:
I put the colors that UX team used in the design, but you're right!

Reply:
yeah, we had this issue in the past too. I have adjusted few colors that didn't match existing patterns. These discrepancies happens between converting colors from hex/rgb to HSL so I wouldn't be scared of slightly changing them if nobody can tell the difference anyway.